contracts/mech_usage/MechActivityChecker.sol

```diff
@@ -10,42 +10,42 @@ interface IMultisig {
     function nonce() external view returns (uint256);
 }
 
-// AgentMech interface
-interface IAgentMech {
-    /// @dev Gets the requests count for a specific account.
-    /// @param account Account address.
-    /// @return requestsCount Requests count.
-    function getRequestsCount(address account) external view returns (uint256 requestsCount);
+// Mech Marketplace interface
+interface IMechMarketplace {
+    /// @dev Gets deliveries count for a specific operator.
+    /// @param operator Operator address.
+    /// @return Deliveries count.
+    function getOperatorDeliveriesCount(address operator) external view returns (uint256);
```

Comment:
Operator is annoying here. I'd say "Service" because Operator has another use in Olas.